test/functionalTest/cases/0000_statistics_operation/statistics_with_semaphores.test

```diff
@@ -38,80 +38,49 @@ echo
 echo "+++++ 2. statistics in JSON"
 orionCurl --url /statistics --json
 echo
-
-
-#
-# About
-#   <versionRequests>REGEX((0|1))</versionRequests>   AND
-#   "versionRequests": "REGEX((0|1))"
-# When running under valgrind, an additional "GET /version" is issued to make sure the broker is running
-#
-#
-# About
-#   "subCacheRefreshs": "REGEX((0|1))"   AND
-#   <subCacheRefreshs>REGEX((0|1))</subCacheRefreshs>
-# When the broker is started with -noCache, there are no refreshes performed
-#
+echo
 
 --REGEXPECT--
 +++++ 1. statistics in XML
 HTTP/1.1 200 OK
-Content-Length: REGEX((842|844))
+Content-Length: 91
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <orion>
-  <versionRequests>REGEX((0|1))</versionRequests>
-  <statisticsRequests>1</statisticsRequests>
-  <requestSemaphoreWaitingTime>REGEX(\d+.\d+)</requestSemaphoreWaitingTime>
-  <dbConnectionPoolWaitingTime>REGEX(\d+.\d+)</dbConnectionPoolWaitingTime>
-  <transactionSemaphoreWaitingTime>REGEX(\d+.\d+)</transactionSemaphoreWaitingTime>
-  <subCacheSemaphoreWaitingTime>REGEX(\d+.\d+)</subCacheSemaphoreWaitingTime>
-  <curlContextMutexWaitingTime>REGEX(\d+.\d+)</curlContextMutexWaitingTime>
-  <timeStatSemaphoreWaitingTime>REGEX(\d+.\d+)</timeStatSemaphoreWaitingTime>
-  <uptime_in_secs>REGEX(1?\d)</uptime_in_secs>
-  <measuring_interval_in_secs>REGEX(1?\d)</measuring_interval_in_secs>
-  <subCacheRefreshs>REGEX((0|1))</subCacheRefreshs>
-  <subCacheInserts>0</subCacheInserts>
-  <subCacheRemoves>0</subCacheRemoves>
-  <subCacheUpdates>0</subCacheUpdates>
-  <subCacheItems>0</subCacheItems>
+  <message>XML not supported in statistics operations, use JSON</message>
 </orion>
 
 
 +++++ 2. statistics in JSON
 HTTP/1.1 200 OK
-Content-Length: REGEX((693|695))
+Content-Length: REGEX(\d+)
```

Comment:
It is very costly to struggle with the fine-grain adjustment of Content-Length in this cases, taking into account the no-valgrind vs. valgrind variability. Thus, I'm using REGEX(\d+), assuming that the important part is in the payload itself.